### PR TITLE
[v6r20] Using ObjectLoader for platform discovery

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*WMS
CHANGE: the platform discovery can be VO-specific. 

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
If there's an agreement on the code in this PR, I will look for adding some documentation. Any suggestion where?

**Comment by atsareg** [comment]
What do you want to achieve here ? VO specific or extension specific functions ? This is not the same.

**Comment by atsareg** [comment]
In this PR you use ObjectLoader to load Resources helper object. This is used in order to load if needed extension specific implementation of a class. This has nothing to do with VO specific modules. The only automatic way to resolve some VO specific elements is Operations CS section where you get VO specific parameters in a generic way. But I guess you want to use, for example, LHCb specific Resources helper implementation which is available from the LHCbDIRAC extension.

**Comment by fstagni** [comment]
Yes: I want to use LHCb specific Resources helper implementation (which I still have to code).

File: docs/source/UserGuide/Tutorials/ManagingUserCredentials/index.rst [hunk 1]
```diff
@@ -15,13 +15,9 @@ This section assumes that the DIRAC client is already installed and configured.
 
   - Firefox:
 
-      Preferences -> Advanced -> View Certificates -> Select your certificate -> Backup
+      Preferences -> Provacy & Security -> View Certificates -> Select your certificate -> Backup
```
Comment (andresailer) [inline docs/source/UserGuide/Tutorials/ManagingUserCredentials/index.rst]: Typo: `Privacy`

**Comment by fstagni** [comment]
Review OK?

**Pull request merged**